app/graphql/types/mutation_type.rb

```
@@ -5,8 +5,9 @@
   class MutationType < GraphQL::Schema::Object
     description 'The mutation root of this schema'
 
-    field :update_photo_title, mutation: Mutations::UpdatePhotoTitle
+    field :update_album_title, mutation: Mutations::UpdateAlbumTitle
```

rubocop (1.69.1)

[convention] Missing field description

(GraphQL/FieldDescription)